controllers/events.js

```diff
@@ -96,10 +97,16 @@ const joinEvent = async (req, res) => {
   const payload = { roomId, userId, role };
```

Comment:
we will not need roomId  from body now because we're getting from the API itself.
So send that roomId in payload